schema/src/main/java/org/hl7/v3/AD.java

```diff
@@ -261,6 +263,25 @@
     @XmlAttribute(name = "isNotOrdered")
     protected Boolean isNotOrdered;
 
+    public List<String> getStreetAddressLine() {
```

Comment:
Could you move getters and setters to the bottom of the file (or just below the fields declarations) please?